cpu/atmega_common/periph/spi.c

```diff
@@ -78,10 +78,11 @@ void spi_init_pins(spi_t bus)
 #endif
 }
 
-int spi_acquire(spi_t bus, spi_cs_t cs, spi_mode_t mode, spi_clk_t clk)
+void spi_acquire(spi_t bus, spi_cs_t cs, spi_mode_t mode, spi_clk_t clk)
 {
     (void)bus;
```

Comment:
```suggestion
```

Reply:
Note that if an argument is used by an `assert()` only, it will be unused once the assert is disabled via `NDEBUG`. The reason is that `assert()` is (as required by the C standard) implement as a preprocessor macro.